File: Strivers/src/basicrecursion/gfgntimes/GFGNtimes.java
package basicrecursion.gfgntimes;

import java.util.Scanner;

public class GFGNtimes {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int number = sc.nextInt();
        printGFGNtimes(number);
    }

    private static void printGFGNtimes(int num){
        if (num == 0) return;
        System.out.print("GFG ");
        printGFGNtimes(num - 1);
    }
}

File: Strivers/src/basicrecursion/gfgntimes/GFGNtimes.kt
package basicrecursion.gfgntimes

import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val num = sc.nextInt()
    GFGNtimes(num)
}

fun GFGNtimes(n: Int) {
    if (n == 0) return
    print("GFG ")
    GFGNtimes(n - 1)
}